### DISCUSSION TITLE
Githup education

### DISCUSSION THREAD

**Discussion by Ahmad-Alhindy** [community/community#142363] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have been subbmitted in github education shouldnt I now get access to copilot?

**Comment by buraito-kun** [comment, marked as answer]
You might need to wait around 5 days after being approved for GitHub Education to receive GitHub Pro. Once your GitHub Pro access is activated, you can then use GitHub Copilot.
![image](https://github.com/user-attachments/assets/3e259b6a-944e-45bd-9f90-48191c49f6be)

**Reply by Ahmad-Alhindy** [reply]
Thank you for help

**Comment by pooriafarmayeshi** [comment]
Hello, my academic email belongs to Khwarazmi University, but when I click 'Continue' to check the location, it just stops. What should I do? Thank you very much.
